jujugui/static/gui/src/app/views/topology/service.js

```diff
@@ -498,14 +498,28 @@ YUI.add('juju-topology-service', function(Y) {
     },
 
     /**
+      Get the DOM node if the container has been provided by YUI, otherwise the
+      container will be the DOM node already.
+
+      @method getContainer
+      @param {Object} context The context to get the container for.
+      @return {Object} A DOM node.
+    */
+    getContainer: function(context) {
+      context = context ? context : this;
+      const container = context.get('container');
+      return container.getDOMNode && container.getDOMNode() || container;
+    },
+
+    /**
       Attaches the drag and drop events for this view. These events need to be
       here because attaching them in the events object causes drag and drop
       events to stop bubbling at odd places cross browser.
 
       @method _attachDragEvents
     */
     _attachDragEvents: function() {
-      var container = this.get('container'),
+      var container = Y.one(this.get('container')),
```

Comment:
That's correct, it appears to be able to attach the delegated events with either `Y.one` and `Y.Node` but I've updated it to `Y.Node` anyway.